class ErrorAplicacion(Exception):
    pass
raise ErrorAplicacion("Manejo de errores de la Aplicaci√≥n:")